Sources/AardvarkReveal/RevealServiceBrowser.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  Copyright 2020 Square, Inc.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+final class RevealServiceBrowser: NSObject, NetServiceBrowserDelegate {
+
+    // MARK: - Life Cycle
+
+    override init() {
+        super.init()
+
+        serviceBrowser.delegate = self
+    }
+
+    // MARK: - Internal Properties
+
+    var localService: NetService? {
```

Comment:
This tries to find a Reveal server running on the same device. It doesn't differentiate between multiple Reveal servers running on the same device, which may cause unexpected behavior when running apps in multiple simulators. That said, I don't think filing a bug report from a simulator is a common use case currently, so this probably isn't something we need to worry about immediately.